diff --git a/node_modules/@strapi/admin/admin/src/content-manager/components/RepeatableComponent/DraggedItem/index.js b/node_modules/@strapi/admin/admin/src/content-manager/components/RepeatableComponent/DraggedItem/index.js
index 5b2bde0..bc21e52 100644
--- a/node_modules/@strapi/admin/admin/src/content-manager/components/RepeatableComponent/DraggedItem/index.js
+++ b/node_modules/@strapi/admin/admin/src/content-manager/components/RepeatableComponent/DraggedItem/index.js
@@ -179,7 +179,7 @@ const DraggedItem = ({
     dropRef: drop(dropRef),
   };
 
-  const accordionTitle = toString(displayedValue);
+  const accordionTitle = toString(displayedValue).length > 150 ? toString(displayedValue).slice(0, 150) + '...' : toString(displayedValue);
   const accordionHasError = hasErrors ? 'error' : undefined;
 
   return (
diff --git a/node_modules/@strapi/admin/admin/src/pages/HomePage/ContentBlocks.js b/node_modules/@strapi/admin/admin/src/pages/HomePage/ContentBlocks.js
index a0d0fda..52cf5d3 100644
--- a/node_modules/@strapi/admin/admin/src/pages/HomePage/ContentBlocks.js
+++ b/node_modules/@strapi/admin/admin/src/pages/HomePage/ContentBlocks.js
@@ -1,14 +1,14 @@
 import React from 'react';
 import styled from 'styled-components';
 import { useIntl } from 'react-intl';
+import { NavLink as RouterNavLink } from 'react-router-dom';
 import { ContentBox, useTracking } from '@strapi/helper-plugin';
 import { Stack } from '@strapi/design-system/Stack';
-import InformationSquare from '@strapi/icons/InformationSquare';
-import CodeSquare from '@strapi/icons/CodeSquare';
-import PlaySquare from '@strapi/icons/PlaySquare';
-import FeatherSquare from '@strapi/icons/FeatherSquare';
+import Landscape from '@strapi/icons/Landscape';
+import Folder from '@strapi/icons/Folder';
+import Pencil from '@strapi/icons/Pencil';
 
-const BlockLink = styled.a`
+const BlockLink = styled(RouterNavLink)`
   text-decoration: none;
 `;
 
@@ -23,81 +23,53 @@ const ContentBlocks = () => {
   return (
     <Stack spacing={5}>
       <BlockLink
-        href="https://strapi.io/resource-center"
-        target="_blank"
-        rel="noopener noreferrer nofollow"
-        onClick={() => handleClick('didClickonReadTheDocumentationSection')}
+        to="/content-manager/collectionType/api::post.post"
       >
         <ContentBox
           title={formatMessage({
             id: 'global.documentation',
-            defaultMessage: 'Documentation',
+            defaultMessage: 'News',
           })}
           subtitle={formatMessage({
             id: 'app.components.BlockLink.documentation.content',
             defaultMessage: 'Discover the essential concepts, guides and instructions.',
           })}
-          icon={<InformationSquare />}
+          icon={<Pencil />}
           iconBackground="primary100"
         />
       </BlockLink>
       <BlockLink
-        href="https://strapi.io/starters"
-        target="_blank"
-        rel="noopener noreferrer nofollow"
-        onClick={() => handleClick('didClickonCodeExampleSection')}
+        to="/content-manager/collectionType/api::page.page"
       >
         <ContentBox
           title={formatMessage({
             id: 'app.components.BlockLink.code',
-            defaultMessage: 'Code example',
+            defaultMessage: 'Page',
           })}
           subtitle={formatMessage({
             id: 'app.components.BlockLink.code.content',
             defaultMessage: 'Learn by using ready-made starters for your projects.',
           })}
-          icon={<CodeSquare />}
+          icon={<Folder />}
           iconBackground="warning100"
         />
       </BlockLink>
       <BlockLink
-        href="https://strapi.io/blog/categories/tutorials"
-        target="_blank"
-        rel="noopener noreferrer nofollow"
-        onClick={() => handleClick('didClickonTutorialSection')}
+        to="/plugins/upload"
       >
         <ContentBox
           title={formatMessage({
             id: 'app.components.BlockLink.tutorial',
-            defaultMessage: 'Tutorials',
+            defaultMessage: 'Medialibrary',
           })}
           subtitle={formatMessage({
             id: 'app.components.BlockLink.tutorial.content',
             defaultMessage: 'Follow step-by-step instructions to use and customize Strapi.',
           })}
-          icon={<PlaySquare />}
+          icon={<Landscape />}
           iconBackground="secondary100"
         />
       </BlockLink>
-      <BlockLink
-        href="https://strapi.io/blog"
-        target="_blank"
-        rel="noopener noreferrer nofollow"
-        onClick={() => handleClick('didClickonBlogSection')}
-      >
-        <ContentBox
-          title={formatMessage({
-            id: 'app.components.BlockLink.blog',
-            defaultMessage: 'Blog',
-          })}
-          subtitle={formatMessage({
-            id: 'app.components.BlockLink.blog.content',
-            defaultMessage: 'Read the latest news about Strapi and the ecosystem.',
-          })}
-          icon={<FeatherSquare />}
-          iconBackground="alternative100"
-        />
-      </BlockLink>
     </Stack>
   );
 };
diff --git a/node_modules/@strapi/admin/admin/src/pages/HomePage/SocialLinks.js b/node_modules/@strapi/admin/admin/src/pages/HomePage/SocialLinks.js
index e5cb43b..3136c62 100644
--- a/node_modules/@strapi/admin/admin/src/pages/HomePage/SocialLinks.js
+++ b/node_modules/@strapi/admin/admin/src/pages/HomePage/SocialLinks.js
@@ -3,197 +3,111 @@
  * SocialLink
  */
 
-import React from 'react';
-import styled from 'styled-components';
-import { useIntl } from 'react-intl';
-import { Typography } from '@strapi/design-system/Typography';
-import { Box } from '@strapi/design-system/Box';
-import { Stack } from '@strapi/design-system/Stack';
-import { Grid, GridItem } from '@strapi/design-system/Grid';
-import { LinkButton } from '@strapi/design-system/v2/LinkButton';
-import { Link } from '@strapi/design-system/v2/Link';
-import ExternalLink from '@strapi/icons/ExternalLink';
-import Github from '@strapi/icons/Github';
-import Discord from '@strapi/icons/Discord';
-import Reddit from '@strapi/icons/Reddit';
-import Strapi from '@strapi/icons/Strapi';
-import Twitter from '@strapi/icons/Twitter';
-import Discourse from '@strapi/icons/Discourse';
+ import React from 'react';
+ import styled from 'styled-components';
+ import { useIntl } from 'react-intl';
+ import { Typography } from '@strapi/design-system/Typography';
+ import { Box } from '@strapi/design-system/Box';
+ import { Stack } from '@strapi/design-system/Stack';
+ import { Grid, GridItem } from '@strapi/design-system/Grid';
+ import { LinkButton } from '@strapi/design-system/v2/LinkButton';
 
-const StyledDiscord = styled(Discord)`
-  path {
-    fill: #7289da !important;
-  }
-`;
+ const socialLinks = [
+   {
+     name: 'Website',
+     link: 'https://mpuik.vercel.app/',
+     alt: 'website',
+   },
+   {
+     name: 'Telegram',
+     link: 'https://t.me/MPCC_ChNU',
+     alt: 'telegram',
+   },
+   {
+     name: 'Facebook',
+     link: 'https://www.facebook.com/mpcc.dpt',
+     alt: 'fb',
+   },
+   {
+     name: 'Instagram',
+     link: 'https://www.instagram.com/mpuik_chnu',
+     alt: 'instagram',
+   }
+ ];
 
-const StyledReddit = styled(Reddit)`
-  > path:first-child {
-    fill: #ff4500;
-  }
-`;
-const StyledStrapi = styled(Strapi)`
-  > path:first-child {
-    fill: #8e75ff;
-  }
-  > path:nth-child(2) {
-    fill: #8e75ff;
-  }
-  > path:nth-child(3) {
-    fill: #8e75ff;
-  }
-`;
+ const LinkCustom = styled(LinkButton)`
+   display: flex;
+   align-items: center;
+   border: none;
 
-const StyledTwitter = styled(Twitter)`
-  path {
-    fill: #1da1f2 !important;
-  }
-`;
+   svg {
+     width: ${({ theme }) => theme.spaces[6]};
+     height: ${({ theme }) => theme.spaces[6]};
+   }
 
-const StyledDiscourse = styled(Discourse)`
-  > path:first-child {
-    fill: #231f20;
-  }
-  > path:nth-child(2) {
-    fill: #fff9ae;
-  }
-  > path:nth-child(3) {
-    fill: #00aeef;
-  }
-  > path:nth-child(4) {
-    fill: #00a94f;
-  }
-  > path:nth-child(5) {
-    fill: #f15d22;
-  }
-  > path:nth-child(6) {
-    fill: #e31b23;
-  }
-`;
+   span {
+     word-break: keep-all;
+   }
+ `;
 
-const socialLinks = [
-  {
-    name: 'Github',
-    link: 'https://github.com/strapi/strapi/',
-    icon: <Github fill="#7289DA" />,
-    alt: 'github',
-  },
-  {
-    name: 'Discord',
-    link: 'https://slack.strapi.io/',
-    icon: <StyledDiscord />,
-    alt: 'discord',
-  },
-  {
-    name: 'Reddit',
-    link: 'https://www.reddit.com/r/Strapi/',
-    icon: <StyledReddit />,
-    alt: 'reddit',
-  },
-  {
-    name: 'Twitter',
-    link: 'https://twitter.com/strapijs',
-    icon: <StyledTwitter />,
-    alt: 'twitter',
-  },
-  {
-    name: 'Forum',
-    link: 'https://forum.strapi.io',
-    icon: <StyledDiscourse />,
-    alt: 'forum',
-  },
-  {
-    name: 'Blog',
-    link: 'https://strapi.io/blog?utm_source=referral&utm_medium=admin&utm_campaign=career%20page',
-    icon: <StyledStrapi />,
-    alt: 'blog',
-  },
-  {
-    name: 'We are hiring!',
-    link: 'https://strapi.io/careers?utm_source=referral&utm_medium=admin&utm_campaign=blog',
-    icon: <StyledStrapi />,
-    alt: 'career',
-  },
-];
+ const GridGap = styled(Grid)`
+   row-gap: ${({ theme }) => theme.spaces[2]};
+   column-gap: ${({ theme }) => theme.spaces[4]};
+ `;
 
-const LinkCustom = styled(LinkButton)`
-  display: flex;
-  align-items: center;
-  border: none;
+ const SocialLinks = () => {
+   const { formatMessage } = useIntl();
 
-  svg {
-    width: ${({ theme }) => theme.spaces[6]};
-    height: ${({ theme }) => theme.spaces[6]};
-  }
+   return (
+     <Box
+       as="aside"
+       aria-labelledby="join-the-community"
+       background="neutral0"
+       hasRadius
+       paddingRight={5}
+       paddingLeft={5}
+       paddingTop={6}
+       paddingBottom={6}
+       shadow="tableShadow"
+     >
+       <Box paddingBottom={7}>
+         <Stack spacing={5}>
+           <Stack spacing={3}>
+             <Typography variant="delta" as="h2" id="join-the-community">
+               {formatMessage({
+                 id: 'app.components.HomePage.community',
+                 defaultMessage: 'Join the community',
+               })}
+             </Typography>
+             <Typography textColor="neutral600">
+               {formatMessage({
+                 id: 'app.components.HomePage.community.content',
+                 defaultMessage:
+                   'Discuss with team members, contributors and developers on different channels',
+               })}
+             </Typography>
+           </Stack>
+         </Stack>
+       </Box>
+       <GridGap>
+         {socialLinks.map(socialLink => {
+           return (
+             <GridItem col={6} s={12} key={socialLink.name}>
+               <LinkCustom
+                 size="L"
+                 variant="tertiary"
+                 href={socialLink.link}
+                 isExternal
+               >
+                 {socialLink.name}
+               </LinkCustom>
+             </GridItem>
+           );
+         })}
+       </GridGap>
+     </Box>
+   );
+ };
 
-  span {
-    word-break: keep-all;
-  }
-`;
+ export default SocialLinks;
 
-const GridGap = styled(Grid)`
-  row-gap: ${({ theme }) => theme.spaces[2]};
-  column-gap: ${({ theme }) => theme.spaces[4]};
-`;
-
-const SocialLinks = () => {
-  const { formatMessage } = useIntl();
-
-  return (
-    <Box
-      as="aside"
-      aria-labelledby="join-the-community"
-      background="neutral0"
-      hasRadius
-      paddingRight={5}
-      paddingLeft={5}
-      paddingTop={6}
-      paddingBottom={6}
-      shadow="tableShadow"
-    >
-      <Box paddingBottom={7}>
-        <Stack spacing={5}>
-          <Stack spacing={3}>
-            <Typography variant="delta" as="h2" id="join-the-community">
-              {formatMessage({
-                id: 'app.components.HomePage.community',
-                defaultMessage: 'Join the community',
-              })}
-            </Typography>
-            <Typography textColor="neutral600">
-              {formatMessage({
-                id: 'app.components.HomePage.community.content',
-                defaultMessage:
-                  'Discuss with team members, contributors and developers on different channels',
-              })}
-            </Typography>
-          </Stack>
-          <Link href="https://feedback.strapi.io/" isExternal endIcon={<ExternalLink />}>
-            {formatMessage({
-              id: 'app.components.HomePage.roadmap',
-              defaultMessage: 'See our road map',
-            })}
-          </Link>
-        </Stack>
-      </Box>
-      <GridGap>
-        {socialLinks.map(socialLink => {
-          return (
-            <GridItem col={6} s={12} key={socialLink.name}>
-              <LinkCustom
-                size="L"
-                startIcon={socialLink.icon}
-                variant="tertiary"
-                href={socialLink.link}
-                isExternal
-              >
-                {socialLink.name}
-              </LinkCustom>
-            </GridItem>
-          );
-        })}
-      </GridGap>
-    </Box>
-  );
-};
-
-export default SocialLinks;
