diff --git a/node_modules/@strapi/admin/admin/src/pages/HomePage/ContentBlocks.js b/node_modules/@strapi/admin/admin/src/pages/HomePage/ContentBlocks.js
index 9feb7bb..3d53a87 100644
--- a/node_modules/@strapi/admin/admin/src/pages/HomePage/ContentBlocks.js
+++ b/node_modules/@strapi/admin/admin/src/pages/HomePage/ContentBlocks.js
@@ -3,10 +3,9 @@ import styled from 'styled-components';
 import { useIntl } from 'react-intl';
 import { ContentBox, useTracking } from '@strapi/helper-plugin';
 import { Stack } from '@strapi/design-system/Stack';
-import InformationSquare from '@strapi/icons/InformationSquare';
-import CodeSquare from '@strapi/icons/CodeSquare';
-import PlaySquare from '@strapi/icons/PlaySquare';
-import FeatherSquare from '@strapi/icons/FeatherSquare';
+import Pencil from '@strapi/icons/Pencil';
+import Folder from '@strapi/icons/Folder';
+import Landscape from '@strapi/icons/Landscape';
 
 const BlockLink = styled.a`
   text-decoration: none;
@@ -23,10 +22,7 @@ const ContentBlocks = () => {
   return (
     <Stack spacing={5}>
       <BlockLink
-        href="https://strapi.io/resource-center"
-        target="_blank"
-        rel="noopener noreferrer nofollow"
-        onClick={() => handleClick('didClickonReadTheDocumentationSection')}
+        href="/dashboard/content-manager/collectionType/api::post.post"
       >
         <ContentBox
           title={formatMessage({
@@ -37,15 +33,12 @@ const ContentBlocks = () => {
             id: 'app.components.BlockLink.documentation.content',
             defaultMessage: 'Discover the essential concepts, guides and instructions.',
           })}
-          icon={<InformationSquare />}
+          icon={<Pencil />}
           iconBackground="primary100"
         />
       </BlockLink>
       <BlockLink
-        href="https://strapi.io/starters"
-        target="_blank"
-        rel="noopener noreferrer nofollow"
-        onClick={() => handleClick('didClickonCodeExampleSection')}
+        href="/dashboard/content-manager/collectionType/api::page.page"
       >
         <ContentBox
           title={formatMessage({
@@ -56,15 +49,12 @@ const ContentBlocks = () => {
             id: 'app.components.BlockLink.code.content',
             defaultMessage: 'Learn by using ready-made starters for your projects.',
           })}
-          icon={<CodeSquare />}
+          icon={<Folder />}
           iconBackground="warning100"
         />
       </BlockLink>
       <BlockLink
-        href="https://strapi.io/blog/categories/tutorials"
-        target="_blank"
-        rel="noopener noreferrer nofollow"
-        onClick={() => handleClick('didClickonTutorialSection')}
+        href="/dashboard/plugins/upload"
       >
         <ContentBox
           title={formatMessage({
@@ -75,29 +65,10 @@ const ContentBlocks = () => {
             id: 'app.components.BlockLink.tutorial.content',
             defaultMessage: 'Follow step-by-step instructions to use and customize Strapi.',
           })}
-          icon={<PlaySquare />}
+          icon={<Landscape />}
           iconBackground="secondary100"
         />
       </BlockLink>
-      <BlockLink
-        href="https://strapi.io/blog"
-        target="_blank"
-        rel="noopener noreferrer nofollow"
-        onClick={() => handleClick('didClickonBlogSection')}
-      >
-        <ContentBox
-          title={formatMessage({
-            id: 'app.components.BlockLink.blog',
-            defaultMessage: 'Blog',
-          })}
-          subtitle={formatMessage({
-            id: 'app.components.BlockLink.blog.content',
-            defaultMessage: 'Read the latest news about Strapi and the ecosystem.',
-          })}
-          icon={<FeatherSquare />}
-          iconBackground="alternative100"
-        />
-      </BlockLink>
     </Stack>
   );
 };
